README.md

```diff
@@ -0,0 +1,33 @@
+# Core Snap
+
+This repository contains the official Core snap. The core snap plays
+
+## Reporting Issues
+
+Please report all issues on the Launchpad project page
+https://bugs.launchpad.net/snapd/+filebug
+
+## Building
+
+To build the gadget snap locally please use `sudo snapcraft`.
```

Comment:
@niemeyer yes, sudo is needed since it pulls in a bunch of stacked chroots for live-build which is designed for lp-buildd operation.